src/c/struktury/struktury.md

```diff
@@ -1,131 +1,220 @@
 # Struktury
-Tato sekce je ve výstavbě 🚧.
-
-<!--Již umíme pracovat s jednoduchými datovými typy, které nám mohou
-reprezentovat celá nebo reálná čísla, či řetězce znaků. Práce se
-strukturovanými datovými typy můžeme do jisté míry považovat za
-předstupeň k objektově orientovanému programování.
-
-## Co jsou struktury
+**Struktury** (*structures*) nám umožňují popsat nový datový typ, který se bude skládat z
+jednoho či více tzv. **členů** (*members*)[^1]. Každému členu musíme určit jeho jméno a datový typ.
+Novou strukturu můžeme popsat pomocí tzv. *deklarace struktury*:
+```c
+struct <název struktury> {
```

Comment:
Mozna bych dal jeste poznaku, ze nazvy struktur budeme chti pojmenovavat [Pascal Case]( c/promenne/pojmenovavani.html#víceslovné-názvy)

Reply:
Přemýšlel jsem, jestli tam mít sekci o pojmenovávání no, vlastně moc nevím, co je nejčastější konvence. Ale pro UPR asi můžeme použít Pascal Case no, už to máme ve skriptech nadefinované.